server/preferences.go

```diff
@@ -3,39 +3,35 @@ package main
 import (
 	"fmt"
 
+	"github.com/mattermost/mattermost-plugin-msteams/server/store/storemodels"
 	"github.com/mattermost/mattermost/server/public/model"
 )
 
 const (
 	PreferenceCategoryPlugin = "pp_" + pluginID
```

Comment:
Couldn't move this one in the storemodels as it depends on `pluginID`. I actually wonder if it's an issue or not, if we change the pluginID for some reason all the preferences will be lost.